src/components/Logo/AssetLogo.tsx

```diff
@@ -0,0 +1,60 @@
+import { SupportedChainId } from 'constants/chains'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { MissingImageLogo } from './BaseLogo'
+
+const LogoImage = styled.img<{ size: string }>`
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  background: radial-gradient(white 60%, #ffffff00 calc(70% + 1px));
+  border-radius: 50%;
+  -mox-box-shadow: 0 0 1px white;
+  -webkit-box-shadow: 0 0 1px white;
+  box-shadow: 0 0 1px white;
```

Comment:
Did you intentionally remove the border?

Reply:
Yeah, I wasn't sure what the border was doing here (I'm not familiar with the purpose of 0px borders), but it added extra contrast to the new white shadow I added that I didn't think was necessary